posts/2024/2024_06_12_kaustav_week_2.rst

```diff
@@ -0,0 +1,56 @@
+My Journey Continues: Week 2 Progress with DIPY
+===============================================
+
+.. post:: June 12 2024
```

Comment:
The `post` date is the date when you wrote the blog. I believe you did both 2 and 3 on the same day.